dallinger/experiment_server/experiment_server.py

```diff
@@ -550,6 +550,21 @@ def create_participant(worker_id, hit_id, assignment_id, mode):
     defined in reference to the participant object. You must specify the
     worker_id, hit_id, assignment_id, and mode in the url.
     """
+    fingerprint_hash = request.args.get('fingerprint_hash')
+    try:
+        fingerprint_found = models.Participant.query.\
+            filter_by(fingerprint_hash=fingerprint_hash).one_or_none()
+    except NoResultFound:
```

Comment:
Does this work with `.one_or_none()`? My understanding is that `.one()` will throw an exception if no or multiple results are found, but that `.one_or_none.()` returns either the lone result or None, or throws an exception, in which case the `except NoResultFound` will never get hit. Is that right?